components/GameCard.js

```diff
@@ -0,0 +1,40 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function GameCard({ game }) {
+  return (
+    <StyledCard>
+      <StyledTitle>{game.name.value}</StyledTitle>
+      <StyledRankTitle>{game.rank}</StyledRankTitle>
+      <Image
+        src={game.thumbnail.value}
+        alt={game.name.value}
+        width={100}
+        height={100}
+      ></Image>
```

Comment:
please use a self closing tag here :
```suggestion
      />
```